server/server.go

```diff
@@ -145,14 +146,19 @@ func (s *Server) getRoleMapping(IP string) (*mappings.RoleMappingResult, error)
 	return roleMapping, nil
 }
 
-// HealthResponse represents a response for the health check.
-type HealthResponse struct {
+// healthResponse represents a response for the health check.
```

Comment:
These changes seem to be unrelated to the iptable change on exit, it would have been better to keep that separated. Is the goal to add a 5s timeout on the metadata check?

Reply:
Yeah exactly - encountered some funky behaviour as a result of the lack of a timeout so bundled the change up, but happy to move if needed though!